client/README.md

```diff
@@ -0,0 +1,447 @@
+
+### Commands
+
+The fastest way to learn more about `cook` is with the `-h` (or `—help`) option.
+
+```
+usage: cook [-h] [--cluster CLUSTER] [--config CONFIG] [--dry-run] [--no-dry-run]
+                   {create-job,create-group,kill-job,kill-instance,retry-job,await-job,await-instance,query-job,query-instance,query-group}
+                   ...
+
+positional arguments:
+  {create-job,create-group,kill-job,kill-instance,retry-job,await-job,await-instance,query-job,query-instance,query-group}
+    create-job          create job for command
+    create-group        create group
+    kill-job            kill job related to uuid
+    kill-instance       kill instance related to uuid
+    retry-job           retry job related to uuid
+    await-job           await job related to uuid
+    await-instance      await instance related to uuid
+    query-job           query job related to uuid
+    query-instance      query instance related to uuid
+    query-group         query group related to uuid
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --cluster CLUSTER, -c CLUSTER
+                        cluster to use
+  --config CONFIG, -C CONFIG
+                        config file to use
+  --dry-run, -d         print action and arguments without submitting
+  --no-dry-run
+
+```
+
+All global options (`—cluster`, `—config`, `—dry-run`) can be provided when using subcommands. `—dry-run` is useful for testing commands without actually attempting to submit.
+
+In cases where a sub-command can take multiple positional arguments, it’s possible to pass a single dash (`-`) instead. This will cause `cook` to read input from `stdin`, with each line corresponding to one positional argument. In this way, you can pipe the output from one command into the input of another. It also makes it possible to operate on multiple entities at once. For instance, you could start multiple jobs by piping one command per line to `cook create-job`.
+
+All options can also be provided from a file by prefixing the file path with an `@` sign. For instance, to start a job with a command read from a file: `cook create-job @command.txt`
+
+#### `create-job`
+
+You can create one or more jobs with `create-job`. The command is the only required option, and can be provided as a positional argument, via `stdin`, or from a file. Jobs can also be passed as “raw” JSON data by setting the `—raw` flag. This command returns UUIDs, one per line, for each created job.
```

Comment:
It'll submit to the default `cluster`... if you were using something like the config file toward the top of the README, it would be the `dev0` cluster.